client/client-multi/client-objective-c/KaaTests/channel/DefaultChannelManagerTest.m

```diff
@@ -226,7 +229,7 @@ - (void)testSingleBootstrapServerFailed {
     
     [verifyCount(failoverManager, times(1)) onServerChangedWithConnectionInfo:anything()];
     
-    [channelManager onServerFailedWithConnectionInfo:server];
+    [channelManager onServerFailedWithConnectionInfo:server failoverStatus:FAILOVER_STATUS_CURRENT_BOOTSTRAP_SERVER_NA];
```

Comment:
Text alignment here is weird, either put it on the same line or indent second line.